x/incentives/keeper/distribute_test.go

```diff
@@ -569,9 +606,15 @@ func (s *KeeperTestSuite) TestDistribute_ExternalIncentives_NoLock() {
 				incentivesEpochDuration := s.App.IncentivesKeeper.GetEpochInfo(s.Ctx).Duration
 				incentivesEpochDurationSeconds := osmomath.NewDec(incentivesEpochDuration.Milliseconds()).QuoInt(osmomath.NewInt(1000))
 
+				// If uptime is not authorized, we expect to fall back to default
+				expectedUptime := types.DefaultConcentratedUptime
+				if tc.authorizedUptime {
+					expectedUptime = tc.distrTo.Duration
+				}
```

Comment:
nit: suggest hardcoding the expected uptime in the test case struct as dynamically selecting the expected value in the test case logic is prone to test configuration bugs, especially, as the test case grows